packages/app/components/drop/drop-form.tsx

```diff
@@ -142,7 +144,7 @@ export const DropForm = () => {
   const windowWidth = useWindowDimensions().width;
 
   const [accordionValue, setAccordionValue] = useState("");
-  const { clearStorage } = usePersistDropForm(FORM_DATA_KEY, {
+  const { clearStorage } = usePersistForm(FORM_DATA_KEY, {
```

Comment:
rename `usePersistDropForm` to `usePersistForm`, because it also can use in other forms I think.

Reply:
💯